docs/conceptual/iommu.rst

```diff
@@ -0,0 +1,55 @@
+.. meta::
+   :description: Input-Output Memory Management Unit (IOMMU)
+   :keywords: IOMMU, DMA, PCIe, xGMI, AMD, ROCm
+
+****************************************************************
+Input-Output Memory Management Unit (IOMMU)
+****************************************************************
+
+The I/O Memory Management Unit (IOMMU) provides memory remapping services for I/O devices. It adds support for address translation and system memory access protection on direct memory access (DMA) transfers from peripheral devices. 
```

Comment:
Instead of "It adds support" should it be something like "It supports address translation..."